core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/runtimeRoutesModels.scala

```diff
@@ -34,7 +34,6 @@ object RuntimeConfigRequest {
   final case class GceConfig(
     machineType: Option[MachineTypeName],
     diskSize: Option[DiskSize],
-    region: Option[ZoneName] = None,
```

Comment:
This was subtle: an extraneous `region` param in `RuntimeConfigRequest.GceConfig`. Because both `ZoneName` and `RegionName` encode from strings, and each has default arguments, I think circe was getting confused and populating `region` instead of `zone`, which had no effect.